lib/toolbox/saved_struct.c

```diff
@@ -0,0 +1,149 @@
+#include "saved_struct.h"
+#include <furi.h>
+#include <stdint.h>
+#include <storage/storage.h>
+
+#define SAVED_STRUCT_TAG    "SAVED_STRUCT"
+
+typedef struct {
+    uint8_t magic;
+    uint8_t version;
+    uint8_t checksum;
+    uint8_t flags;
+    uint32_t timestamp;
```

Comment:
doesn't make much sense here. If application needs timestamp it can have it in its own structure.

Reply:
So, finally,
drop timestamp and flags. Ok?